src/styles/search.css

```diff
@@ -0,0 +1,35 @@
+.search-input {
+  width: 30%;
+  height: 30px;
+  font-size: 20px;
+  border: 1px solid black;
+  float: left;
+  border-top-left-radius: 10px;
+  border-bottom-left-radius: 10px;
+  font-family: "Roboto Mono", monospace;
+}
+.search-form {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+.search-button {
+  margin: 0;
+  float: left;
+  width: 5%;
```

Comment:
This is causing the text in the button to be cropped at smaller screen sizes, for a small element like this you could easily just set it to a static width and avoid this